### DISCUSSION TITLE
DuckDB Support

### DISCUSSION THREAD

**Discussion by nirlevi** [jeremyevans/sequel#2243] [General]
Hello @jeremyevans , 
first , Thanks for the great work! 
are there any plans with adding support for duckdb 
Thanks

**Comment by jeremyevans** [comment]
There are no current plans to support DuckDB in Sequel.  However, it's not too difficult to build external Sequel database adapters, so if you'd like to access DuckDB through Sequel, I recommend creating an external adapter.
